uaclient/cli.py

```diff
@@ -813,12 +818,16 @@ def action_enable(args, *, cfg, **kwargs):
             ret &= ent_ret
         except exceptions.UserFacingError as e:
             print(e)
+            ret = False
```

Comment:
@lucasmoura @blackboxsw @renanrodrigo How do you all feel about this line I added? It causes `ua enable`s that fail to exit non-zero. I added it when initially debugging the new ros integration test, but it has broken several other tests. I can either change those tests to expect non-zero exits, or I can remove this line - what do you think?

Reply:
I think this makes sense, if `ua enable` fails for some reason we should not have a non-zero return